tests/geometry/region.cpp

```diff
@@ -55,63 +55,39 @@ std::ostream& operator<<(std::ostream& os, const wxRegion& rgn)
 // test class
 // ----------------------------------------------------------------------------
 
-class RegionTestCase : public CppUnit::TestCase
-{
-public:
-    RegionTestCase() { }
-
-private:
-    CPPUNIT_TEST_SUITE( RegionTestCase );
-        CPPUNIT_TEST( Validity );
-        CPPUNIT_TEST( Intersect );
-    CPPUNIT_TEST_SUITE_END();
-
-    void Validity();
-    void Intersect();
-
-    wxDECLARE_NO_COPY_CLASS(RegionTestCase);
-};
-
-// register in the unnamed registry so that these tests are run by default
-CPPUNIT_TEST_SUITE_REGISTRATION( RegionTestCase );
-
-// also include in its own registry so that these tests can be run alone
-CPPUNIT_TEST_SUITE_NAMED_REGISTRATION( RegionTestCase, "RegionTestCase" );
-
-void RegionTestCase::Validity()
+TEST_CASE("Validity", "[region]")
 {
     wxRegion r;
 
-    CPPUNIT_ASSERT_MESSAGE
+    CHECK_FALSE
     (
-        "Default constructed region must be invalid",
-        !r.IsOk()
+        // Default constructed region must be invalid
```

Comment:
Minor, but I don't think it makes sense to keep comments inside `CHECK`s: messages had to be there because they were used by the macro, but this is ignored by `CHECK`, so could just as well move it above it to make the check shorter and more readable.